### PR TITLE
Fix: hahealth: return fail when report failed (bsc#1231840)

### PR DESCRIPTION
**Pull request by aleksei-burlakov**
This will help external applications (e.g. hawk) better understand the result of the 'crm script run health' execution.

### CODE REVIEW

File: scripts/health/hahealth.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def create_report():
 
 
 if not create_report():
-    crm.exit_ok({'status': 'Failed to create report'})
+    crm.exit_fail({'status': 'Failed to create report'})
```
Comment (nicholasyang2022) [inline scripts/health/hahealth.py:25]: [`exit_fail`](https://github.com/ClusterLabs/crmsh/blob/76fb6a0be7c121e971907d98af7e4b3bbc6f9548/utils/crm_script.py#L60) does not accept a dictonary.

```suggestion
    crm.exit_fail('Failed to create report')
```

**Review by nicholasyang2022 — COMMENTED** [review]
And should we also change [this line](https://github.com/ClusterLabs/crmsh/blob/6fa784dcb26be957e02118b7b14303c3f31a5398/scripts/health/hahealth.py#L34)?

File: scripts/health/hahealth.py (hunk 1 above)
Comment (aleksei-burlakov) [reply scripts/health/hahealth.py:25]: > And should we also change [this line](https://github.com/ClusterLabs/crmsh/blob/6fa784dcb26be957e02118b7b14303c3f31a5398/scripts/health/hahealth.py#L34)?

That's right.

> [exit_fail](https://github.com/ClusterLabs/crmsh/blob/76fb6a0be7c121e971907d98af7e4b3bbc6f9548/utils/crm_script.py#L60) does not accept a dictionary.

Actually it does. I think, it was in the older versions of python, you couldn't pass a dictionary to a `print` function. Now it's a valid code `print({'status': 'Failed to create report'})`

**Review by nicholasyang2022 — APPROVED** [review]

**Comment by nicholasyang2022** [comment]
@liangxin1300 We have test failures.

**Pull request merged**